Source/Core/Core/PowerPC/JitCommon/JitCache.cpp

```diff
@@ -70,7 +90,18 @@ void JitBaseBlockCache::Clear()
 
   valid_block.ClearAll();
 
-  fast_block_map.fill(nullptr);
+  if (bm_fd >= 0) {
+      munmap(fast_block_map, 0x7FFFFFFFF);
+      ftruncate(bm_fd, 0);
+      ftruncate(bm_fd, 0x7FFFFFFFF);
+  }
+
+  if (bm_fd < 0) {
+    bm_fd = CreateFileMapping("dolphin-pc", 0x7FFFFFFFF);
+  }
+
+  fast_block_map = (JitBlock**)mmap64(nullptr, 0x7FFFFFFFF, PROT_READ | PROT_WRITE,
```

Comment:
How did you arrive at 0x7'FFFF'FFFF? In the JITs, you're shifting a 32-bit value left by 1, so I would have imagined you would need a size of 0x2'0000'0000.

Reply:
Brain = Broken